src/api/script/api_test_in_spec.sh

```diff
@@ -8,6 +8,7 @@ MYSQL_DATADIR=$MYSQL_BASEDIR/data
 MEMCACHED_PID_FILE=$TEMP_DIR/memcached.pid
 MYSQL_SOCKET_DIR=`mktemp -d`
 MYSQL_SOCKET=$MYSQL_SOCKET_DIR/mysql.socket
+RETRY=1
```

Comment:
@hennevogel like discussed in https://github.com/openSUSE/open-build-service/issues/4826 we disabled rspec-retry for CI. We only use it now it the package builds (where it is really painful). This way we can spot flaky tests and actually fix them.

cc @bgeuken